src/ssl.c

```diff
@@ -5004,6 +5004,16 @@ int AddCA(WOLFSSL_CERT_MANAGER* cm, DerBuffer** pDer, int type, int verify)
 static int wolfSSL_RAND_InitMutex(void);
 #endif
 
+#if defined(OPENSSL_EXTRA) && defined(HAVE_ATEXIT)
+static void AtExitCleanup(void)
+{
+    int i = initRefCount;
```

Comment:
Why not set initRefCount to 1 and call wolfSSL_Cleanup once?

Reply:
Updated to your suggestion.

Reply:
Should the `if (initRefCount == 0) {` check be `> 0`? Guessing this code should only be called if the initRefCount is greater than zero.

Reply:
The cleanup routine will never decrement it to lower than 0 but I suppose this would bug proof this a little.